src/components/AdministrationPage/AdministrationPage.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState } from 'react';
+import {
+  Button,
+  Divider,
+  Flex,
+  FlexItem,
+  Form,
+  FormGroup,
+  PageSection,
+  PageSectionVariants,
+  Switch,
+  Text,
+  TextContent,
+  TextArea,
+  TextInput,
+} from '@patternfly/react-core';
+
+export const AdministrationPage = () => {
```

Comment:
Please add some paddings to the first button and maintenance switcher, so that it looks like something like:
![image](https://user-images.githubusercontent.com/17566891/127985527-864d5ef0-a93c-4655-98ef-0813e944f1f2.png)